### PR TITLE
Psalm suggested fixes.

### PR DESCRIPTION
**Pull request by dereuromark**
As per Psalm
```
Psalm can automatically fix 31 of these issues.
Run Psalm again with 
--alter --issues=MissingClosureReturnType,InvalidReturnType --dry-run
to see what it can fix.
```

do we want to add those?

### CODE REVIEW

File: src/ORM/Marshaller.php [hunk 1]
```diff
@@ -106,13 +106,19 @@ protected function _buildPropertyMap(array $data, array $options): array
...
             }
             if (isset($options['isMerge'])) {
-                $callback = function ($value, EntityInterface $entity) use ($assoc, $nested) {
+                $callback = function (
+                    $value,
+                    EntityInterface $entity
+                ) use (
+                    $assoc,
+                    $nested
+                ): array|EntityInterface|null {
```
Comment (LordSimal) [inline src/ORM/Marshaller.php:115]: i think this formatting can be improved

Comment (dereuromark) [reply src/ORM/Marshaller.php:115]: How? Any different and CS fails (too long line length) etc.

Comment (LordSimal) [reply src/ORM/Marshaller.php:115]: I thought maybe we can put parts of that function definition in 1 line but it seems we can't...

**Comment by markstory** [comment]
Looks good to me.

**Pull request merged**